docs-ui/components/alert.stories.js

```diff
@@ -27,6 +28,29 @@ storiesOf('UI|Alerts/Alert', module)
       </Grid>
     ))
   )
+  .add(
+    'With icons',
+    withInfo('Inline alert messages')(() => (
+      <Grid>
+        <Alert type="info" icon={<IconInfo size="md" />}>
```

Comment:
do we really need to allow a dynamic icon in Alert? I'd display the icon only according to the type

Reply:
I've seen us use a variety of icons in alerts. I think yes.

Reply:
What if we infer icon from type - most of the times you would want IconInfo for "info", IconCheckmark for "success", etc. anyway.
And we could provide a way to override/provide your own icon for those few special cases (or hide icon altogether).

Reply:
Yeah we already do this in PanelAlert actually, I'm fine with that idea. But would rather not do it here. I just added this story so I could see what these look like with icons.